File: trunk/esper/src/main/java/net/esper/eql/core/StreamNotFoundException.java
package net.esper.eql.core;

/**
 * Exception to indicate that a stream name could not be resolved.
 */
public class StreamNotFoundException extends StreamTypesException
{
    /**
     * Ctor.
     * @param msg - message
     */
    public StreamNotFoundException(String msg)
    {
        super(msg);
    }
}

File: NEsper/eql/core/StreamNotFoundException.cs
using System;
namespace net.esper.eql.core
{
	
	/// <summary> Exception to indicate that a stream name could not be resolved.</summary>
	[Serializable]
	public class StreamNotFoundException:StreamTypesException
	{
		/// <summary> Ctor.</summary>
		/// <param name="msg">- message
		/// </param>
		public StreamNotFoundException(String msg):base(msg)
		{
		}
	}
}